onmt/inputters/inputter.py

```diff
@@ -706,6 +706,32 @@ def __iter__(self):
                 return
 
 
+class Tracker(object):
```

Comment:
Perhaps Tracker is a bit of a vague name? Suggestions: DataTracker of DataProgressTracker.